native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -610,194 +761,129 @@ EMSCRIPTEN_BINDINGS(spine) {
 
     class_<AtlasRegion>("TextureAtlasRegion")
         //.property("page", &AtlasRegion::page)
-        .function("getName", optional_override([](AtlasRegion &obj) {
-            return STRING_SP2STD((const String)obj.name); }))
+        .function("getName", optional_override([] (AtlasRegion &obj) { return obj.name; }))
         .property("x", &AtlasRegion::x)
         .property("y", &AtlasRegion::y)
         .property("index", &AtlasRegion::index)
         .property("rotate", &AtlasRegion::rotate)
         .property("degrees", &AtlasRegion::degrees);
         //.property("texture", &AtlasRegion::height)
 
+    class_<TextureLoader>("TextureLoader");
+        
+
     class_<Atlas>("TextureAtlas")
         .constructor<const String &, TextureLoader *, bool>()
-        .function("findRegion", optional_override([](Atlas &obj, const std::string &name) {
-            return obj.findRegion(STRING_STD2SP(name)); }), allow_raw_pointers());
+        .function("findRegion", &Atlas::findRegion, allow_raw_pointers());
 
     class_<PowInterpolation, base<Interpolation>>("Pow")
-        .constructor<int>()
-        .function("apply", &PowInterpolation::apply);
```

Comment:
apply is a virtual function in base class `Interpolation`, no need to bind it in sub class.